--- udp.go	2018-04-16 20:33:26.166432005 +0200
+++ udp-new.go	2018-04-16 20:36:26.975514289 +0200
@@ -114,7 +114,7 @@
 func NewRUDP(conn net.PacketConn, logger log.Logger) *RUDP {
 	t := &RUDP{
 		conn:       conn,
-		nextPktID:  uint56(generator.Intn(maxUint56 + 1)),
+		nextPktID:  uint56(generator.Int63n(maxUint56 + 1)),
 		readEvents: make(chan *readEventDesc, maxReadEvents),
 		closedChan: make(chan struct{}),
 		doneChan:   make(chan struct{}),
